### PR TITLE
Update access-management.md

### PR DESCRIPTION
**Pull request by yanivhershkovich**
Based on a recent query from a customer and after doing a few tests, suggesting changes to the 'GCP Marketplace team management' section.
I suggest having these changes reviewed by PM and/or R&D to confirm them.

### CODE REVIEW

File: content/rc/administration/access-management.md [hunk 1]
```diff
@@ -128,19 +128,8 @@ If you subscribed to Redis Enterprise Cloud using Google Cloud Platform (GCP) Ma
 
 To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign the following roles to the user:
 
-- To designate a viewer, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.viewer`
-- To designate an owner, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.admin`
+- An 'Owner' or 'Editor' role will allow the user to access the Redis console and perform changes.
+- A 'Viewer' role will allow the user to access the Redis console but not to make any changes.
+- A user with a 'Browser' role will not be able to access the Redis console.
```
Comment (lanceleonard) [inline content/rc/administration/access-management.md:133]:
```suggestion
- A user with a 'Browser' role cannot access the Redis admin console.
```

File: content/rc/administration/access-management.md [hunk 2]
```diff
@@ -128,19 +128,8 @@ If you subscribed to Redis Enterprise Cloud using Google Cloud Platform (GCP) Ma
 
 To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign the following roles to the user:
 
-- To designate a viewer, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.viewer`
-- To designate an owner, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.admin`
+- An 'Owner' or 'Editor' role will allow the user to access the Redis console and perform changes.
+- A 'Viewer' role will allow the user to access the Redis console but not to make any changes.
```
Comment (lanceleonard) [inline content/rc/administration/access-management.md:132]:
```suggestion
- A 'Viewer' role lets the user access the Redis admin console but not make changes.
```

File: content/rc/administration/access-management.md [hunk 3]
```diff
@@ -128,19 +128,8 @@ If you subscribed to Redis Enterprise Cloud using Google Cloud Platform (GCP) Ma
 
 To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign the following roles to the user:
 
-- To designate a viewer, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.viewer`
-- To designate an owner, assign `serviceusage.serviceUsageViewer` & `redisenterprisecloud.admin`
+- An 'Owner' or 'Editor' role will allow the user to access the Redis console and perform changes.
```
Comment (lanceleonard) [inline content/rc/administration/access-management.md:131]:
```suggestion
- 'Owner' and 'Editor' roles let users access the Redis admin console and make changes.
```

**Review by lanceleonard — APPROVED** [review]
LGTM.  Provisionally approved (with edits & pending PM approval).

**Comment by rrelledge** [comment]
Closing this PR without merging as this info is now outdated. PR #2324 removes this section since it is no longer needed for the latest GCP Marketplace listing.

**Pull request closed without merging**